src/v/cluster/partition.h

```diff
@@ -103,7 +103,7 @@ class partition {
             return _rm_stm->last_stable_offset();
         }
 
-        return raft::details::next_offset(_raft->last_stable_offset());
```

Comment:
should we delete / modify the large comment above? seems to be now some what out of date?